src/partials-js/history_city.js

```diff
@@ -41,10 +41,11 @@ function GenerateViewHistory(arrayOfKey, reload) {
     historyArray = `
     <a class="history_button_link" href="${el}">
       ${loadStorage(el).city.name}
-    </a>`;
+    </a>
+    <a class="history_button_delete" href="${el}">X</a>`;
     history.innerHTML = historyArray;
 
-    historySection.append(history);
+    http: historySection.append(history);
```

Comment:
po co tutaj ten protokół http?

Reply:
już zmieniłem, przez przypadek to mi sie musiało wkleić